sdk/LATEST

```diff
@@ -1,4 +1,4 @@
-9bd7e7ade7d2222cf95cca5fa6c611369ee1b89e 3.4.0-snapshot.20251002.14252.0.v9bd7e7ad SPLIT_RELEASE
+9345440318cd1b1e3c8363b97925cf0d52768e14 3.4.0-snapshot.20251007.14268.0.v93454403 SPLIT_RELEASE
 a77a3d96abde9c36c56d42ae4e2b8fec88f7ee35 3.4.0-adhoc.20250828.14144.0.va77a3d96 SPLIT_RELEASE
```

Comment:
```suggestion
9bd7e7ade7d2222cf95cca5fa6c611369ee1b89e 3.4.0-snapshot.20251002.14252.0.v9bd7e7ad SPLIT_RELEASE
a77a3d96abde9c36c56d42ae4e2b8fec88f7ee35 3.4.0-adhoc.20250828.14144.0.va77a3d96 SPLIT_RELEASE
```

You'll want to overwrite the "adhoc" line instead of the "snapshot" line here - the adhoc codepath is a bit underspecified.

Reply:
we use the terminology of "adhoc release" for release of a commit which does not belong to a release branch.
the `release.sh` will add use the `adhoc`  keyword instead of  snapshot in this case.